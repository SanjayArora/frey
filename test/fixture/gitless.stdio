--> Frey version '0.3.9'
--> Will run: [ 'init', 'config', 'prepare', 'validate' ]
--> {hostname} - init
--> {hostname} - config
--> {hostname} - prepare
Creating private key '{tmpdir}/frey-gitless/frey-frey-gitless.pem'
Creating public key '{tmpdir}/frey-gitless/frey-frey-gitless.pub'
Creating private encrypted key '{tmpdir}/frey-gitless/frey-frey-gitless.pem.cast5'
Found 'terraform' with version '0.6.12'
Found 'terraformInventory' with version '0.6.0'
Found 'pip' with version {{global_pip_version}}
Found 'pyhcl-0.1.15' with version '0.1.15'
Found 'pyhcl-0.2.0' with version '0.2.0'
Found 'ansible' with version '2.0.0'
--> {hostname} - validate

--> Exiting with error: Frey requires project (and state) to be under Git, and residu to be ignored.
