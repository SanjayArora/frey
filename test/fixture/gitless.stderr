
--> Exiting with error: Error while checking for '{root}/test/scenario/gitless/.git'
--> Details:

undefined
